frontend/apps/picasso/defi/polkadot/hooks/useStakingRewards.ts

```diff
@@ -38,17 +36,14 @@ export const useStakingRewards = () => {
   const isStakingPositionsLoadingState = useStore(
     (state) => state.isStakingPositionsLoadingState
   );
-  const { data, loading, error, refetch } = useQuery<StakingPositions>(
-    GET_STAKING_POSITIONS,
-    {
-      variables: {
-        accountId: account?.address,
-      },
-      pollInterval: 30000,
-    }
-  );
+  const { data, loading } = useQuery<StakingPositions>(GET_STAKING_POSITIONS, {
+    variables: {
+      accountId: account?.address,
+    },
+    pollInterval: 30000,
+  });
 
-  const assetId = pica.picassoId?.toNumber() || 1;
+  const assetId = pica.chainId.picasso?.toNumber() || 1;
```

Comment:
we should avoid vanilla js numbers for indexing, wdyt?

Reply:
There is no way around this at the moment, because `pica.chainId.picasso` might be null, the fallback to this, `parachainApi?.consts.assets.nativeCurrencyId` is also `u128 | undefined`.

So, there is either, go down all the way on all components, or stop it here.

Reply:
@easteregg no no, i meant instead of `number` type lets use a `string`

Reply:
@easteregg unrelated to this PR but we can make this improvement later